README.md

```diff
@@ -180,8 +183,8 @@ class ExampleSerializer(actorSystem: ExtendedActorSystem)
   extends CirceAkkaSerializer[MySerializable](actorSystem) {
     // ...
     override lazy val codecs = Seq(Register[CommandOne]) // WHOOPS someone forgot to register CommandTwo...
-    // ... but Codec Registration Checker will throw an error here!
-    // ...
+    // ... but Codec Registration Checker will throw a compilation error here:
+    // `No codec for `CommandOne` is registered in class annotated with @org.virtuslab.ash.annotation.Serializer`
 }
```

Comment:
@MarconZet do you think it's worthwhile mentioning the pdf somehow from the README?

Reply:
(docs/presentation.pdf, added by this PR)

Reply:
I think that after sphere-it we will add a section with the talk, and put the reference there.